client-rest/README.md

```diff
@@ -1,9 +1,9 @@
-![Arrow Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
+![Aetheric Banner](https://github.com/aetheric-oss/.github/blob/main/assets/readme-banner.png)
```

Comment:
use `raw` instead of `blob`

Reply:
https://github.com/aetheric-oss/svc-contact/pull/10/commits/716a5407b4bf47dd6fd513153b56a69d3c6da709